modules/sspBCBidAdapter.md

```diff
@@ -28,11 +35,12 @@ var adUnits = [
       bids: [{
          bidder: 'sspBC',
          params: {
-            id: '006',                          // required
-            siteId: '235911',                   // required
+            id: '006',                          // optional
```

Comment:
These params should return a response from your server. I tried using these in a test page and received a `204 No Content` response.

Reply:
@msm0504 Could You check with the following values
`{
                "bidder": "sspBC",
                "params": {
                    "id": "003",
                    "siteId": "235860",
                    "test": 1
                }
            }`

If it helps - I've updated our test pages:
https://bdr.wpcdn.pl/tests/bidderv4/test-adapter_friendly.html
https://bdr.wpcdn.pl/tests/bidderv4/test-adapter_google.html

Both are using above config, with static CMP2 module, and current version of prebid (4.20.pre) containing adapter, consent management and currency module (built using  gulp build-bundle-dev --modules=sspBCBidAdapter,consentManagement,currency).

Also, please note that we've been testing 4.7 adapter on O&O inventory; as a result of that, I've pushed small fixes today (see  https://github.com/prebid/Prebid.js/pull/6118/commits/18237abe5045494cef960f2be88e1c712b2555b2)